app/routes.py

```diff
@@ -67,6 +67,9 @@ def posts():
 @login_required # allows only a logged in user to access account page
 def feed():
     job_posts = Post.query.all()
+    form = FeedApplyForm()
+    if form.validate_on_submit():
+        application = Application(user_id = current_user.id, post_id = )
     return render_template("FeedPage.html", title = 'Feed',  posts = job_posts )
```

Comment:
for rendering the feed page what value is the posts variable supposed to be set to? currently we have 2 for loops in the HTML Jinja template. One loops through "posts" (this variable has the dynamic styling applied to it) the other loops through job_posts (which I think was just a variable used to test the page when it was first set up).  I can add this to the issues feed if needed.